src/main.rs

```diff
@@ -123,7 +125,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
 
             let response = result.saveToDb(&conn)?;
 
-            println!("( {} records )", result.phone_number_location.len());
+            println!("( {} records )", 1);
```

Comment:
It always printing a fixed value.

Reply:
Because there is only one content in the file. Am I wrong ?

Reply:
@GawenMonnot Thanks for your answer. You can print like this ` println!("1 records")` and if the result is always one content, you don't need to print out IMHO.

Reply:
Yes you are right. Thank you!